### PR TITLE
Add get_stats endpoint.

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: datacommons/examples/places.py [hunk 1]
```diff
@@ -40,12 +40,16 @@ def main():
...
+  stats = dc.get_stats(['geoId/05', 'geoId/06'], 'dc/0hyp6tkn18vcb')
+  print(stats)
+
   # Get related places.
-  print('Get related places')
-  # TODO(*): s/Cenus/Census/g when data is ready in BT.
-  related_places = dc.get_related_places(['geoId/06085'], 'Person', 'count',
-      'CenusACS5yrSurvey', "measuredValue", {"gender": "Female"})
-  print(related_places)
+#  print('Get related places')
```
Comment (pradh) [inline datacommons/examples/places.py:49]: why is this commented?

File: datacommons/places.py [hunk 2]
```diff
@@ -71,7 +71,72 @@ def get_places_in(dcids, place_type):
...
+
+  Examples:
+    We would like to get the :obj:`TimeSeries` of the number of males
+    at least 25 years old that attended 12th grade but did not receive
+    a high school diploma
+    (`dc/0hyp6tkn18vcb <https://browser.datacommons.org/kg?dcid=dc/0hyp6tkn18vcb>`_)
+    in `Arkansas <https://browser.datacommons.org/kg?dcid=geoId/05>`_
+    and `California <https://browser.datacommons.org/kg?dcid=geoId/06>`_.
+
+    >>> get_stats(["geoId/05", "geoId/06"], "dc/0hyp6tkn18vcb")
```
Comment (pradh) [inline datacommons/places.py:101]: Mahesh says the rollout is complete, could you use a human-curated DCID in this example?

(Adding a TODO in this PR is fine, if it needs more work.)

File: datacommons/test/places_test.py [hunk 3]
```diff
@@ -142,5 +206,53 @@ def test_no_dcids(self, urlopen):
...
+    """
+    # Set the API key
+    dc.set_api_key('TEST-API-KEY')
+
+    # Call get_places_in with one dcid that does not exist
+    bad_dcids_1 = dc.get_stats(['geoId/05', 'dc/MadDcid'], 'dc/0hyp6tkn18vcb')
+    self.assertDictEqual(bad_dcids_1, {
+        'geoId/06085': ['geoId/0649670'],
+        'dc/MadDcid': []
+    })
```
Comment (pradh) [inline datacommons/test/places_test.py:239]: I'm puzzled by the checks. Why don't the expected dict contain time-series?

Also s/get_places_in/get_stats/

Comment (tjann) [reply datacommons/test/places_test.py]: Sorry, I prematurely requested your review. Was having trouble testing locally so Bo had me create a PR for test to run on CloudRun, shouldn't have requested, though. 

This should be fixed now, but need to see test results.

File: datacommons/examples/places.py (hunk 1 above)
Comment (tjann) [reply datacommons/examples/places.py:50]: It was broken on current master, I've added a TODO.

**Pull request merged**